src/main/java/json/OpenTriviaDatabase.java

```diff
@@ -0,0 +1,11 @@
+package json;
+
+import io.reactivex.Observable;
+import retrofit2.http.GET;
+
+public interface OpenTriviaDatabase {
+
+    @GET("/api.php?amount=1&category=9&difficulty=easy&type=multiple")
```

Comment:
I assume at some point you can change category and difficulty?

Reply:
Should I change the API call to any category and any difficulty?